bb/ramfs.go

```diff
@@ -115,23 +114,29 @@ func ramfs() {
 
 	paths[filepath.Join(config.Gopath, "src/github.com/u-root/u-root/bb/bbsh")] = []string{"init", "ubin"}
 
-	if *extraPaths != "" {
-		extras := strings.Split(strings.TrimSpace(*extraPaths), " ")
-		for _, x := range extras {
-			p := strings.Split(x, ":")
-			if len(p) != 2 {
-				p = append([]string{"/"}, p...)
+	if *extraCmds != "" {
+		copyc := strings.Fields(*extraCmds)
+		for _, eachPath := range copyc {
+			// eachPath must be an absolute filepath for os.Stat
+			statval, err := os.Stat(eachPath)
```

Comment:
Did you test this command with a string containing a :? Below, you split eachPath on the :. But here, you stat the path with no change. If the path contains a :, then the stat will fail. I think a change this far reaching might need to include a test.